### ISSUE TITLE
Replace juicy-filedrop with starcounter-upload

### ISSUE THREAD

**Issue by warpech** [StarcounterApps/Images#50]
Right now the file [`juicy-filedrop.html`](https://github.com/StarcounterPrefabs/Images/blob/develop/src/Images/wwwroot/Images/elements/so-juicy-filedrop/juicy-filedrop.html) pretends to be https://github.com/juicy/juicy-filedrop.

But actually it is a very modified fork of it, that looks closer to https://github.com/Starcounter/starcounter-upload

We need to add missing features to https://github.com/Starcounter/starcounter-upload and use that instead of [juicy-filedrop.html](https://github.com/StarcounterPrefabs/Images/blob/develop/src/Images/wwwroot/Images/elements/so-juicy-filedrop/juicy-filedrop.html).

@diliger can you do that after #49?

**Comment by aloscha** [comment]
Should I increase starcounter-upload with files extension?

**Comment by warpech** [comment]
There are more differences than that. This needs to be done after #49, so that it is certain what changes are needed.

cc @miyconst

**Comment by aloscha** [comment]
@diliger the #49 is now closed. Can you do it now?

**Comment by warpech** [comment]
@diliger is now busy with Launcher/Website improvements, so this will have to wait.

**Comment by mmnosek** [comment]
Guys, it can't wait that long. I've discovered that it's now impossible to upload an image if a page is loaded over `https`. Can we proceed somehow? :) `starcounter-upload` supports `https` while `juicy-filedrop` does not. I think that there is no reason to make separate issue and fix in `juicy-filedrop` if the plan is to replace it anyway.

**Comment by aloscha** [comment]
I'll look at it on weekend or in beginning of next week.

**Comment by mmnosek** [comment]
ping

**Comment by mmnosek** [comment]
Seems to be fixed. Closing.
